asyncscope.md

```diff
@@ -607,6 +608,78 @@ class Camera {
 }
 ```
 
+## Recursively spawning work until completion
+Below are two ways you could recursively spawn work on a scope using `let_with_async_scope` or `counting_scope`. Both
+examples are perfectly valid ways to ensure that all new work spawned from previous work will complete.
```

Comment:
I'm a bit thrown off by "perfectly valid" here; is there an objection you're trying to preempt with this comment?